generators/server/templates/gradle/_profile_dev.gradle

```diff
@@ -32,19 +32,16 @@ task gulpConstantDev(type: GulpTask) {
     args = ["ngconstant:dev", "--no-notification"]
 }<% } %>
 
-task setProdProperties(dependsOn: bootRun) << {
-    doFirst {
-        System.setProperty('spring.profiles.active', profiles)
-    }
-}
-
 processResources {
+    filesMatching('**/.build-properties.xml') {
+        filter {
+            it.replace('@spring.profiles.default@', profiles)
+        }
+    }
     filesMatching('**/logback-spring.xml') {
         filter {
             it.replace('@logback.loglevel@', logbackLoglevel)
         }
     }
 }
 
-<% if (!skipClient) { %>
-setProdProperties.dependsOn gulpConstantDev<% } %>
```

Comment:
@deepu105 I think you're right, I did this because the `setProdProperties` task is no longer there... But now the indirect dependency to `gulpConstantDev` is gone. However, should that not be a switch, based on active profile, to `gulpConstantDev` or `...Prod`?

I am unable to check this out until tomorrow, unfortunately.